src/pages/patientView/clinicalInformation/ClinicalInformationMutationalSignatureTable.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from "react";
+import {ClinicalAttribute, ClinicalData} from "../../../shared/api/generated/CBioPortalAPI";
+import LazyMobXTable, {Column} from "shared/components/lazyMobXTable/LazyMobXTable";
+import {fetchMutationalSignatureData} from "../../../shared/lib/StoreUtils";
+import styles from './style/patientTable.module.scss';
+import {SHOW_ALL_PAGE_SIZE} from "../../../shared/components/paginationControls/PaginationControls";
+import {IMutationalSignature} from "../../../shared/model/MutationalSignature";
+import convertSamplesData, {IConvertedSamplesData} from "./lib/convertSamplesData";
+import {ISampleRow} from "./ClinicalInformationSamplesTable";
+import _ from 'lodash';
+
+export interface IClinicalInformationMutationalSignatureTableProps {
+    data: IMutationalSignature;
+    showTitleBar?: boolean;
```

Comment:
I think those options should have a default? And should be passed to MutationalSignatureTable.